paasta_tools/kubernetes_tools.py

```diff
@@ -4354,6 +4425,20 @@ def get_kubernetes_secret_env_variables(
     return decrypted_secrets
 
 
+def create_temp_exec_token(kube_client: KubeClient, namespace: str, user: str):
+    """Create a short lived token for exec"""
+    audience = "remote-run-" + user
+    service_account = "default"
```

Comment:
This will need to come from the main paasta config. We'll have to create a user that can just exec in to places.